### PR TITLE
first task

### PR DESCRIPTION
**Pull request by farzad80rad**
react console is added.
react version is downgraded from 17.0.2 to 16.3.0 to be compatible with react console.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-);
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./containers/App";
```
Comment (m-salehi-v) [inline src/index.js]: move App.js outside of containers folder -> src/App.js

File: src/containers/App.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  Switch,
+  Route,
+  Link,
+  useRouteMatch,
+  useParams,
+} from "react-router-dom";
+
+function App() {
+  return (
+    <div>
```
Comment (m-salehi-v) [inline src/containers/App.js]: this div element and the one below <Router> is not needed here

File: src/components/terminal/SimpleTerminal.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+import Terminal from "react-console-emulator";
+
+function SimpleTerminal() {
+  const commands = {
```
Comment (amirhnajafiz) [inline src/components/terminal/SimpleTerminal.js]: Create a new file called **Commands.js** and declare the commands init then import into this file and use it.

**Review by amirhnajafiz — COMMENTED** [review]
Please use _fluent_ branch names.

**Pull request merged**